subxt/src/rpc.rs

```diff
@@ -624,11 +538,29 @@ impl<T: Config> Rpc<T> {
         public_key: Bytes,
         key_type: String,
     ) -> Result<bool, BasicError> {
-        let params = &[to_json_value(public_key)?, to_json_value(key_type)?];
+        let params = rpc_params![public_key, key_type];
         Ok(self.client.request("author_hasKey", params).await?)
     }
 }
 
+/// Build WS RPC client from URL
+pub async fn build_ws_client(url: &str) -> Result<RpcClient, RpcError> {
```

Comment:
nit: I'd like this named even shorter, perhaps just `ws_client`. Matter of taste though.